build/config/ios/rules.gni

```diff
@@ -145,7 +145,12 @@ template("ios_app") {
   bin_gen_target_name = target_name + "_bin"
 
   executable(bin_gen_target_name) {
-    libs = [ "UIKit.framework", "QuartzCore.framework", "OpenGLES.framework" ]
+    libs = [
```

Comment:
It's very strange that these go into the default rule instead of per-executable.